src/lib/operators/jit_operator/operators/jit_read_tuple.hpp

```diff
@@ -0,0 +1,116 @@
+#pragma once
+
+#include "jit_abstract_operator.hpp"
+#include "storage/chunk.hpp"
+#include "storage/table.hpp"
+
+namespace opossum {
+
+/* Base class for all column readers.
+ * We need this class, so we can store a number of JitColumnReaders with different template
+ * specializations in a common data structure.
+ */
+class BaseJitColumnReader {
```

Comment:
See the file name discussion above

Reply:
Maybe you don't need a separate file for the base, but at least `JitColumnReader` should have its own.
Or is it only used by `JitReadTuple`? If so, it could be an inner class.